separate settings and manual scan in admin ui

stklcode opened the pull request:
First step for a UI rework. Addresses #22 and is based on the mockups by @00travelgirl00.
This PR does not introduce any new features and does not change our API, s.t. not all features from the mockups are available yet (theme grouping, last scan timestamp, etc.)

Manual theme file scan is not placed in a separate tab.

Settings table uses default WP settings styles and the scan results are presented in a WP-like table structure.
(could be migrated to settings API which would ease real modularization, but not now)

The first column is yet intentionally left empty. In a followup step I'd like to see grouping the files by theme (or maybe file system hierarchy), so toggle controls should be placed there.

<details>
  <summary>Preview (animated)</summary>

  ![settings tab](https://user-images.githubusercontent.com/12963621/114276094-7a39ae00-9a25-11eb-995f-c0471075d9b3.png)
  ![scan tab](https://user-images.githubusercontent.com/12963621/114275570-55dcd200-9a23-11eb-8295-c6881fb2b23d.gif)
</details>

stklcode committed:
separate settings and manual scan in admin ui

Manual theme file scan is not placed in a separate tab.
Settings table uses default WP settings styles and the scan results
are presented in a WP-like table structure.

Zodiac1978 commented:
This looks great!

Just one idea: Maybe we could add a brighter version of the highlight color as a background-color for the row to make it easier to see what line is processed and how is the result (like on the plugins page).

For example:

```
.av-status-ok {
  background-color: #edfaef !important;
}
```

stklcode commented:
Good idea. We should darken the font color to the 70% values though, to keep a good contrast ratio and override the default opacity of the code-box, s.t. it's not a redish gray.

Like so:
<img src="https://user-images.githubusercontent.com/12963621/118136645-9fdf1c00-b404-11eb-9890-0c7c194f09e1.png" height="250">

stklcode committed:
background-color for result rows

stklcode committed:
add check-column class to prevent auto-width on mobile devices

stklcode committed:
use gray 70 instead of gray 60 for text in results table

stklcode committed:
place labels next to checkboxes on settings page

Shorten the row header and add descriptive labels directly after
the checkboxes to enable certain checks.

stklcode commented:
Rebased on top of the latest _develop_ state.

Tested on a real-world 5.9.3 site without any issues. Still looks like the previously attached screenshot.

stklcode committed:
point link for manual scan to scan-tab directly

Zodiac1978 reviewed (approved):
I do not check every single line, so this needs more eyes to check after merging.
LGTM so far.
